src/SDKs/Kusto/Management.Kusto/generated/DataConnectionsOperations.cs

```diff
@@ -23,20 +23,20 @@ namespace Microsoft.Azure.Management.Kusto
     using System.Threading.Tasks;
 
     /// <summary>
-    /// EventHubConnectionsOperations operations.
```

Comment:
The renaming of this folder to "generated" from "Generated" for some of these files is causing issues when checking out these files. On windows it complains that there are changes and on I bet on unix it will have issues because these files will not be actually used. I think we should move these files back into the "Generated" folder.

Reply:
submitted fix with https://github.com/Azure/azure-sdk-for-net/pull/5281.